Automattic/wp-calypso
# Add p2 network

**Built on top of https://github.com/Automattic/wp-calypso/pull/42155**

#### Changes proposed in this Pull Request

- Team section was renamed to A8C since it was used for A8C Followed
- Add a P2 Network Reader section which should include user's wp4teams sites.
![Markup 2020-05-12 at 9 44 31 PM](https://user-images.githubusercontent.com/2129455/82329716-6854b400-99ea-11ea-8ec7-da8ad3f8d843.png)


#### Testing instructions
- Apply D43646-code
- Apply D43648-code
- Apply D43649-code
- Apply D43808-code
- Test Reader P2 Network unseen status and mark as seen/unseen work as expected
